Sorry for the delay. Here are the internal errors, @kvignesh1420 can you please verify ?

Traceback (most recent call last):
  File "<embedded stdlib>/unittest/case.py", line 59, in testPartExecutor
    yield
  File "<embedded stdlib>/unittest/case.py", line 605, in run
    testMethod()
  File "/third_party/py/absl/testing/parameterized.py", line 282, in bound_param_test
    return test_method(self, **testcase_params)
  File "/third_party/tensorflow/python/framework/test_combinations.py", line 315, in decorated
    execute_test_method()
  File "/third_party/tensorflow/python/framework/test_combinations.py", line 298, in execute_test_method
    test_method(**kwargs_to_pass)
  File "/third_party/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py", line 93, in testCompression
    compressed = compression_ops.compress(element)
  File "/third_party/tensorflow/python/data/experimental/ops/compression_ops.py", line 35, in compress
    tensor_list = structure.to_tensor_list(element_spec, element)
  File "/third_party/tensorflow/python/data/util/structure.py", line 377, in to_tensor_list
    element_spec, element)
  File "/third_party/tensorflow/python/data/util/structure.py", line 326, in _to_tensor_list_helper
    reduce_fn, zip(nest.flatten(element_spec), nest.flatten(element)), [])
  File "/third_party/tensorflow/python/data/util/structure.py", line 323, in reduce_fn
    return encode_fn(state, spec, component)
  File "/third_party/tensorflow/python/data/util/structure.py", line 376, in <lambda>
    lambda state, spec, component: state + spec._to_tensor_list(component),
  File "/third_party/tensorflow/python/ops/ragged/ragged_tensor.py", line 2342, in _to_tensor_list
    return [value._to_variant(batched_input=False)]
  File "/third_party/tensorflow/python/ops/ragged/ragged_tensor.py", line 1929, in _to_variant
    self.nested_row_splits, self.flat_values, batched_input, name)
  File "/third_party/tensorflow/python/ops/gen_ragged_conversion_ops.py", line 406, in ragged_tensor_to_variant
    name=name, ctx=_ctx)
  File "/third_party/tensorflow/python/ops/gen_ragged_conversion_ops.py", line 448, in ragged_tensor_to_variant_eager_fallback
    attrs=_attrs, ctx=ctx, name=name)
  File "/third_party/tensorflow/python/eager/execute.py", line 75, in quick_execute
    raise e
  File "/third_party/tensorflow/python/eager/execute.py", line 60, in quick_execute
    inputs, attrs, num_outputs)
TypeError: An op outside of the function building code is being passed
a "Graph" tensor. It is possible to have Graph tensors
leak out of the function building context by including a
tf.init_scope in your function building code.
For example, the following function will fail:
  @tf.function
  def has_init_scope():
    my_constant = tf.constant(1.)
    with tf.init_scope():
      added = my_constant * 2
The graph tensor has name: RaggedConstant/Const:0

--- a/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
+++ b/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
@@ -28,6 +28,7 @@ from tensorflow.python.framework import combinations
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.ops.ragged import ragged_factory_ops
 from tensorflow.python.platform import test
+from tensorflow.python.framework import test_util
 
 
 def _test_objects():
@@ -60,15 +61,15 @@ def _test_objects():
   ]
 
 
-def _test_eager_only_objects():
+def _test_ragged_based_objects():
   return [
-      combinations.NamedObject(
-          "ragged",
-          ragged_factory_ops.constant([[0, 1, 2, 3], [4, 5], [6, 7, 8], [9]])
+      dict(
+          test_name="ragged",
+          test_input=ragged_factory_ops.constant([[0, 1, 2, 3], [4, 5], [9]])
       ),
-      combinations.NamedObject(
-          "sparse_ragged_structured",
-          {
+      dict(
+          test_name="sparse_ragged_structured",
+          test_input={
               "sparse": sparse_tensor.SparseTensorValue(
                   indices=[[0, 0], [1, 2]], values=[1, 2], dense_shape=[3, 4]),
               "ragged": ragged_factory_ops.constant([[0, 1, 2, 3], [9]])
@@ -81,10 +82,7 @@ class CompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
   @combinations.generate(
       combinations.times(
           test_base.default_test_combinations(),
-          combinations.combine(element=_test_objects())) +
-      combinations.times(
-          test_base.eager_only_combinations(),
-          combinations.combine(element=_test_eager_only_objects())))
+          combinations.combine(element=_test_objects())))
   def testCompression(self, element):
     element = element._obj
 
@@ -96,10 +94,7 @@ class CompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
   @combinations.generate(
       combinations.times(
           test_base.default_test_combinations(),
-          combinations.combine(element=_test_objects())) +
-      combinations.times(
-          test_base.eager_only_combinations(),
-          combinations.combine(element=_test_eager_only_objects())))
+          combinations.combine(element=_test_objects())))
   def testDatasetCompression(self, element):
     element = element._obj
 
@@ -109,6 +104,27 @@ class CompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
     dataset = dataset.map(lambda *x: compression_ops.compress(x))
     dataset = dataset.map(lambda x: compression_ops.uncompress(x, element_spec))
     self.assertDatasetProduces(dataset, [element])
+
+
+class RaggedCompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
+
+  def testCompression(self):
+    for test_case in _test_ragged_based_objects():
+      element = test_case["test_input"]
+      compressed = compression_ops.compress(element)
+      uncompressed = compression_ops.uncompress(
+          compressed, structure.type_spec_from_value(element))
+      self.assertValuesEqual(element, self.evaluate(uncompressed))
+
+  def testDatasetCompression(self):
+    for test_case in _test_ragged_based_objects():
+      element = test_case["test_input"]
+      dataset = dataset_ops.Dataset.from_tensors(element)
+      element_spec = dataset.element_spec
+
+      dataset = dataset.map(lambda *x: compression_ops.compress(x))
+      dataset = dataset.map(lambda x: compression_ops.uncompress(x, element_spec))
+      self.assertDatasetProduces(dataset, [element])
 
 
 if __name__ == "__main__":

--- a/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
+++ b/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
@@ -105,10 +105,7 @@ class CompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
     dataset = dataset.map(lambda x: compression_ops.uncompress(x, element_spec))
     self.assertDatasetProduces(dataset, [element])
 
-
-class RaggedCompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase):
-
-  def testCompression(self):
+  def testRaggedCompression(self):
     for test_case in _test_ragged_based_objects():
       element = test_case["test_input"]
       compressed = compression_ops.compress(element)
@@ -116,7 +113,7 @@ class RaggedCompressionOpsTest(test_base.DatasetTestBase, parameterized.TestCase
           compressed, structure.type_spec_from_value(element))
       self.assertValuesEqual(element, self.evaluate(uncompressed))
 
-  def testDatasetCompression(self):
+  def testRaggedDatasetCompression(self):
     for test_case in _test_ragged_based_objects():
       element = test_case["test_input"]
       dataset = dataset_ops.Dataset.from_tensors(element)

--- a/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
+++ b/tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py
@@ -28,7 +28,6 @@ from tensorflow.python.framework import combinations
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.ops.ragged import ragged_factory_ops
 from tensorflow.python.platform import test
-from tensorflow.python.framework import test_util
 
 
 def _test_objects():

